ansible/roles/test/tasks/advanced-reboot.yml

```diff
@@ -154,85 +183,13 @@
 
       when: new_sonic_image | default('') | length > 0
 
-    - include: ptf_runner.yml
-      vars:
-        ptf_test_name: Advanced-reboot test
-        ptf_test_dir: ptftests
-        ptf_test_path: advanced-reboot.ReloadTest
-        ptf_platform: remote
-        ptf_platform_dir: ptftests
-        ptf_qlen: 1000
-        ptf_test_params:
-        - verbose=False
-        - dut_username=\"{{ ansible_ssh_user }}\"
-        - dut_hostname=\"{{ ansible_host }}\"
-        - reboot_limit_in_seconds={{ reboot_limit }}
-        - reboot_type=\"{{ reboot_type }}\"
-        - portchannel_ports_file=\"/tmp/portchannel_interfaces.json\"
-        - vlan_ports_file=\"/tmp/vlan_interfaces.json\"
-        - ports_file=\"/tmp/ports.json\"
-        - dut_mac='{{ ansible_Ethernet0['macaddress'] }}'
-        - dut_vlan_ip='192.168.0.1'
-        - default_ip_range='192.168.0.0/16'
-        - vlan_ip_range=\"{{ minigraph_vlan_interfaces[0]['subnet'] }}\"
-        - lo_v6_prefix=\"{{ minigraph_lo_interfaces | map(attribute='addr') | ipv6 | first | ipsubnet(64) }}\"
-        - arista_vms=\"['{{ vm_hosts | list | join("','") }}']\"
+    - include: ptf_runner_helper.yml
```

Comment:
`ptf_runner_helper.yml` I'd better rename it.
Because the helper is not general and tightly connected to reboots only.

Reply:
Also you use it only once. Probably it's better return it to how it was before?

Reply:
The helper is run multiple times based on the preboot list. warmboot-sad.yml  specifies the values for this list. For the happy path, it will just be used once

renaming the helper - ptf_runner_reboot.yml?

Reply:
`ptf_runner_reboot.yml` looks good.

Reply:
renamed